src/main/webapp/app/shared/pipes/artemis-duration.pipe.ts

```diff
@@ -0,0 +1,31 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import * as moment from 'moment';
+
+@Pipe({ name: 'durationTo' })
+export class DurationPipe implements PipeTransform {
+    /**
+     * Calculate the duration of an event from the diff of two dates.
+     * @param startDate The start date time of the event. Must be convertible to moment().
+     * @param endDate The end date time of the event. Must be convertible to moment().
+     * @param format Format of the localized date time. Defaults to 'long'.
+     * @returns string 'HH:mm'
+     */
+    transform(
+        startDate: Date | moment.Moment | string | number | null,
+        endDate: Date | moment.Moment | string | number | null,
+        format: 'short' | 'long' | 'short-date' | 'long-date' | 'time' = 'long',
```

Comment:
You are not using it so I think you can remove it. Also `ArtemisDatePipe` does not apply the format *length* to its time format so you will definitely not need it.